src/main.c

```diff
@@ -1557,6 +1564,13 @@ static gboolean status_start(int argc, char **argv)
 	g_debug("status start");
 	r_exit_status = 0;
 
+#if ENABLE_SERVICE == 1
```

Comment:
Here as well. Also I'd prefer `if (!ENABLE_SERVICE)` as below for better compile coverage.

Reply:
Fine with that.